native/storybook/storybook.js

```diff
@@ -5,12 +5,12 @@ import { getStorybookUI, configure } from '@storybook/react-native';
 
 // import stories
 configure(() => {
-  // require('./stories');
   require('../packages/react-native-bpk-component-text/stories');
+  require('../packages/react-native-bpk-component-button/stories');
 }, module);
 
 // This assumes that storybook is running on the same host as your RN packager,
 // to set manually use, e.g. host: 'localhost' option
-const StorybookUI = getStorybookUI({ port: 7007, onDeviceUI: true });
```

Comment:
This will conflict with Matt's branch which changes it to be `onDeviceUI: false`

Reply:
Now changed.